packages/discord.js/typings/index.d.ts

```diff
@@ -2330,7 +2330,7 @@ export interface DefaultReactionEmoji {
   name: string | null;
 }
 
-export class ForumChannel extends TextBasedChannelMixin(GuildChannel, true, [
+export class BaseForumChannel extends TextBasedChannelMixin(GuildChannel, true, [
```

Comment:
```suggestion
export class ThreadsChannel extends TextBasedChannelMixin(GuildChannel, true, [
```
Or maybe `ThreadsBasedChannel`

Reply:
```suggestion
export class ThreadOnlyChannel extends TextBasedChannelMixin(GuildChannel, true, [
```
How about `ThreadOnlyChannel`, considering discord/discord-api-docs#6232 refers a forum channel and media channel as "thread-only channel" and internal experiment name for the forum channel before its release was "Threads only channel"?

Reply:
Upstream documentation was merged with "thread-only channel", so I went with that one.